easybuild/tools/build_log.py

```diff
@@ -168,6 +168,21 @@ def exception(self, msg, *args):
         raise EasyBuildError(ebmsg, *args)
 
 
+class NonTerminatingLoggerProxy(object):
+    """
+    Monkeypatch an existing `EasyBuildLog` object so that calling
+    `.error()` does not raise an exception and terminate the program.
+    """
+    def __init__(self, logger):
+        self._logger = logger
```

Comment:
Bcc:
Reply-To:
> @riccardomurri should we also include a `logger.deprecated` call in here, to make sure we trash this when we get to making `log.error` behave as expected?

Either that or a FIXME comment.

(But where would you place a `logger.deprecated` call exactly?)